src/QMCWaveFunctions/TrialWaveFunction.h

```diff
@@ -144,8 +144,13 @@ class TrialWaveFunction
   // Finally, the call to resetParameters progates the new values (step 3).
   // The call to checkOutVariables is a prerequisite for resetParameters to set the local values successfully.
 
+  /** extract underlying OptimizableObject references
+   * @param opt_obj_refs aggregated list of optimizable object references
+   */
+  void extractOptimizableObjectRefs(UniqueOptObjRefs& opt_obj_refs);
```

Comment:
"Refs" probably shouldn't be in the name of the function - that puts too much implementation detail in the name.

Reply:
Good suggestions. However, I would defer the change until the PRs in my pockets gets merged and then make the change once.

Reply:
I have changed the API signature in a later PR, opt_obj_refs is a return value directly.